### DISCUSSION TITLE
Consulta fecha forms

### DISCUSSION THREAD

**Discussion by AbrilIX** [Launch-X-Latam/MisionFrontEnd#411] [Q&A]
Hola chicxs! 
Había leído en el teams que había una fecha estimativa para entregar las prácticas y si mal no recuerdo dijeron que sería hoy 18. Tienen idea cómo sería eso de subirlo al formulario o con tener las prácticas en nuestros githubs ya alcanza?
Gracias, 
que tengan un lindo viernes!

**Comment by iam-leo** [comment]
Hola Abril, las prácticas son mas que nada para que tengas en tu Github y puedas mostrarlas en tu portafolio. La fecha del 18 fue mas que nada tentativa para que los que están atrasados se pongan al corriente, seguro esta semana empecemos con las prácticas de backend y pues sería atrasarse demasiado.
